shopfloor_workstation_label_printer/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Shopfloor Workstation Label Printer",
+    "summary": "Adds a label printer configuration to the user and shopfloor workstation.",
+    "version": "14.0.1.0.0",
+    "category": "Tools",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/wms",
```

Comment:
```suggestion
    "website": "https://github.com/OCA/wms",
    "maintainers": ["TDu"],
```
you can add yourself as maintainershere to be able merge fixes in future